config/src/server/mod.rs

```diff
@@ -31,6 +31,9 @@ pub struct Config {
     /// Configuration of the Google 3rd party client
     #[serde(default)]
     pub google: Option<Google>,
+    /// Configuration for Google login
+    #[serde(default)]
+    pub google_login: Option<GoogleLogin>,
```

Comment:
maybe instead of making `google_login` it'd be better to make struct which would include many logins, e.g

```rs
struct Logins {
    google: GoogleLogin,
}
```

Reply:
Done.